setup.cfg

```diff
@@ -0,0 +1,24 @@
+[flake8]
+max_line_length = 120
+ignore =
+    W391  # allow empty line at end of file
+    W503  # break before binary operator - allow either style
+    W504  # break after binary operator - allow either style
+exclude =
+
+[isort]
```

Comment:
Is isort really necessary outside of the Django projects? Don't think such a small task is worth the extra maintenance, dependencies and learning curve.

Reply:
I like it, and get irritated when imports aren't sorted :)